code/modules/mob/living/basic/space_fauna/meteor_heart/meteor_heart.dm

```diff
@@ -8,7 +8,9 @@
 	icon = 'icons/mob/simple/meteor_heart.dmi'
 	icon_state = "heart"
 	icon_living = "heart"
-	mob_biotypes = MOB_ORGANIC
+	sentience_type = SENTIENCE_BOSS
+	mob_biotypes = MOB_ORGANIC|MOB_EPIC
+	faction = list(FACTION_BOSS)
```

Comment:
why should it be friendly with bosses

Reply:
All bosses inherit this code for consistency, otherwise you would have megafauna attack and kill each other.  From my understanding of the ruin PR, this mob was supposed to be a boss-like challenge.